src/profilers/wall-profiler.ts

```diff
@@ -0,0 +1,44 @@
+import {perftools} from '../profile';
+import {serializeWallProfile} from './profile-serializer';
+const wallProfiler = require('bindings')('wall_profiler');
+
+export class WallProfiler {
+  // Average time in microseconds between samples.
+  samplingInterval: number;
+
+  // True when the profiler is actively profiling.
+  private profiling: boolean;
+
+  constructor(samplingInterval: number) {
```

Comment:
Define this way: `constructor(private samplingInterval: number)` and get rid of property declaration above and assignment below.